src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -598,6 +601,14 @@ public Result call() throws IOException {
                         for (Map.Entry<byte[], NavigableSet<byte[]>> entry : get.getFamilyMap()
                             .entrySet()) {
                             family = entry.getKey();
+                            Map<byte[], TimeRange> colFamTimeRangeMap = get.getColumnFamilyTimeRange();
+                            if (colFamTimeRangeMap.size() > 1) {
```

Comment:
we should also report error when setColumnFamilyTimeRange for A column family and get for B column family

Reply:
Already done.